src/runtimes/node/utils/zip.ts

```diff
@@ -48,6 +51,8 @@ interface ZipNodeParameters {
   filename: string
   mainFile: string
   moduleFormat: ModuleFormat
+  name: string
+  repositoryRoot?: string
```

Comment:
we have to pass this than in the CLI, right?

Reply:
Netlify Build is already passing it down to ZISI: https://github.com/netlify/build/blob/a20207ec7cdc1101a875d9411355bd48f12733a3/packages/build/src/plugins_core/functions/zisi.ts#L68